packages/hep-concurrency/package.py

```diff
@@ -71,3 +71,7 @@ def setup_build_environment(self, env):
         env.prepend_path("PATH", os.path.join(prefix, "bin"))
         # Cleanup.
         sanitize_environments(env, "PATH")
+
+    def setup_run_environment(self, run_env):
+        run_env.prepend_path("ROOT_INCLUDE_PATH", self.prefix.include)
+        run_env.append_path("CET_PLUGIN_PATH", self.prefix.lib)
```

Comment:
`hep-concurrency` should not need to adjust these environment variables.  I'd remove `setup_run_environment`.

Reply:
Done